packages/cli/src/constructs/alert-channel.ts

```diff
@@ -51,11 +83,38 @@ export class AlertChannelRef extends Construct {
 }
 
 /**
- * Creates an Alert Channels
+ * Base class for creating alert channels that notify when checks fail or recover.
+ * 
+ * Alert channels define how and when to send notifications when monitoring checks
+ * fail, recover, or enter a degraded state. Checkly supports multiple alert channel types
+ * including email, SMS, Slack, webhooks, and third-party integrations.
  *
- * @remarks
+ * **Plan Limitations:**
```

Comment:
@hlenke I wouldn't have this here. Pricing in TSDocs is too much and most likely won't be updated when pricing changes.